parquet-column/src/test/java/org/apache/parquet/io/api/TestBinary.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Arrays;
 
 import org.apache.parquet.io.api.TestBinary.BinaryFactory.BinaryAndOriginal;
+import org.junit.Ignore;
```

Comment:
yeah, I had added an ignore to allow the rest of the test suite to run while I was stuck on debugging a test here.